Quaderni di storia religiosa medievale.js

```diff
@@ -0,0 +1,274 @@
+{
+	"translatorID": "5db07685-2d24-4b0a-9659-2e6af73e58ba",
+	"label": "Quaderni di storia religiosa medievale",
+	"creator": "Timotheus Kim",
+	"target": "^https:?//www\\.rivisteweb\\.it/(doi|issn)/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
```

Comment:
New translator => inRepository should be false, don't forget file name prefix "ubtue_"

Reply:
file name changed